dbms/src/Storages/DeltaMerge/ColumnFile/ColumnFileTiny.h

```diff
@@ -87,17 +100,29 @@ class ColumnFileTiny : public ColumnFilePersisted
         UInt64 bytes_,
         PageIdU64 data_page_id_,
         const DMContext & dm_context,
+        const IndexInfosPtr & index_infos_ = nullptr,
         const CachePtr & cache_ = nullptr);
 
     Type getType() const override { return Type::TINY_FILE; }
 
     size_t getRows() const override { return rows; }
     size_t getBytes() const override { return bytes; }
 
+    IndexInfosPtr getIndexInfos() const { return index_infos; }
+    bool hasIndex(Int64 index_id) const
```

Comment:
`hasVectorIndex`?

Reply:
`hasIndex` is better since we will support other types of index later.